Fix #4761: Emit `used-before-assignment` where single assignment only made in except blocks (#5402)
PyCQA/pylint: doc/whatsnew/2.13.rst

Other Changes
=============

> * ``used-before-assignment`` now considers that assignments in an except
>   block may not have occurred.

Reviewer: ```suggestion
* ``used-before-assignment`` now assumes that assignments in except blocks
  may not have occurred and warns accordingly.
```
